assets/tailwind.config.js

```diff
@@ -1,80 +1,66 @@
 // See the Tailwind configuration guide for advanced usage
 // https://tailwindcss.com/docs/configuration
 
-const plugin = require("tailwindcss/plugin")
-const fs = require("fs")
-const path = require("path")
+const {
+  colors,
+  content,
+  fontFamily,
+  plugins,
+  safelist
+} = require("mbta_metro");
+const plugin = require("tailwindcss/plugin");
 
 module.exports = {
-    prefix: "tw-",
-    corePlugins: {
-        preflight: false
+  important: true,
+  corePlugins: {
+    preflight: false
+  },
+  blocklist: ["container", "collapse"],
+  content: [
+    ...content,
+    "./js/**/*.js",
+    "./ts/**/*.ts",
+    "./ts/**/*.tsx",
+    "../lib/dotcom_web.ex",
+    "../lib/dotcom_web/**/*.*ex",
+    "../deps/mbta_metro/lib/mbta_metro/**/*.ex"
```

Comment:
You shouldn't need this.

Reply:
This PR is outdated so this line works for this PR, which predates the /component and /live directories in Metro 😓

Reply:
Which I guess means I need to downgrade the npm lib? Umm hmm

Reply:
Ha nope, the Elixir version of Metro being used in this PR predates mbta_metro 0.0.1. I'll be sure to remove this line when I update it in a later PR!